middleware/healthcheck/healthcheck_test.go

```diff
@@ -0,0 +1,118 @@
+package healthcheck
+
+import (
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/gofiber/fiber/v3"
+	"github.com/stretchr/testify/require"
+	"github.com/valyala/fasthttp"
+)
+
+func Test_HealthCheck_Default(t *testing.T) {
+	t.Parallel()
+
+	app := fiber.New()
+	app.Use(New())
+
+	req, err := app.Test(httptest.NewRequest(fiber.MethodGet, "/readyz", nil))
+	require.NoError(t, err)
+	require.NoError(t, err)
```

Comment:
redundant assert

Reply:
after the golangci hints i changed it via search and replace 
thx for the hint, i will remove it